Markdown/MultiMarkdown.sublime-syntax

```diff
@@ -24,5 +25,5 @@ contexts:
           scope: string.unquoted.multimarkdown
     - match: ''
       push:
-        - meta_scope: meta.content.multimarkdown
-        - include: scope:text.html.markdown
+        - - meta_scope: meta.content.multimarkdown
```

Comment:
Is that style a new feature in version 2?

Reply:
No, that was possible before.

Reply:
Yeah, but Will told Thom not to do it for complicated stuff.

Reply:
I know about chaining anonymous contexts via `- - ...`. I did not expect anonymous contexts with only a `meta_scope` derective to be valid.